build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
OK.  Let me have a look at the incompatibilities tomorrow.  If we're going to move to a standardized version, then we need to do the refactoring necessary for all of our parsers.

BTW, I couldn't find the exact combination of IDEA and Grammar-Kit that we used to build with either.  I'm beginning to think that it was IDEA 13.x...